util/desktop/desktop.go

```diff
@@ -0,0 +1,54 @@
+package desktop
+
+import (
+	"bytes"
+	"fmt"
+	"os"
+	"path/filepath"
+	"sync"
+)
+
+var (
+	bbEnabledOnce sync.Once
+	bbEnabled     bool
+)
+
+func BuildBackendEnabled() bool {
+	bbEnabledOnce.Do(func() {
+		home, err := os.UserHomeDir()
+		if err != nil {
+			return
+		}
+		_, err = os.Stat(filepath.Join(home, ".docker", "desktop-build", ".lock"))
```

Comment:
Think we should be able to get the right filename using https://github.com/docker/buildx/blob/341fb65f6f568696ed5e2585eb747aedaed7ee1a/util/confutil/config.go#L23 the docker cli?

Or is that inaccurate for some reason?

Reply:
The desktop build daemon sets the data folder to `$HOME/.docker/desktop-build` and doesn't take into account `DOCKER_CONFIG`.